### PR TITLE
[FIX] mrp(workorder_hr): fix workorder times

### PR DESCRIPTION
**Pull request by ajf-odoo**
Fix workorder times problems:
- sum in workorders list view (displays numeric value rather than mins:secs)
- timers sometimes starts at -00:01
- almost all tablet view actions reset the timer (click burger, ...)
- clicking pause stops the timer at the next second
- seconds are lost in employee timers
- fix employee list display in workorder form view

task: 2985735

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mrp/models/mrp_workcenter.py [hunk 1]
```diff
@@ -184,7 +184,7 @@ def unblock(self):
         if self.working_state != 'blocked':
             raise exceptions.UserError(_("It has already been unblocked."))
         times = self.env['mrp.workcenter.productivity'].search([('workcenter_id', '=', self.id), ('date_end', '=', False)])
-        times.write({'date_end': fields.Datetime.now()})
+        times.write({'date_end': fields.datetime.now()})
```
Comment (Whenrow) [inline addons/mrp/models/mrp_workcenter.py:187]: Could you import datetime in this file instead ?. I find it easier to understand your choice

File: addons/mrp/static/src/widgets/timer.js [hunk 2]
```diff
@@ -24,55 +41,50 @@ export class MrpTimer extends Component {
                 ? this.props.ongoing
                 : this.props.record.data.is_user_working;
 
-        onWillStart(() => this._runTimer());
+        onWillStart(async () => {
+            const addWorkingDuration = 
+                this.props.ongoing !== undefined
+                    ? false
+                    : this.props.record.data.state == "progress";
+            if(addWorkingDuration) {
```
Comment (Whenrow) [inline addons/mrp/static/src/widgets/timer.js:49]:
```suggestion
            if (this.props.ongoing ===undefined && this.props.record.data.state == "progress") {
```
as you don't need the value of addWorkingDuration later

File: addons/mrp/static/src/widgets/timer.js [hunk 3]
```diff
@@ -24,55 +41,50 @@ export class MrpTimer extends Component {
...
+            }
+        }, 1000);
     }
 }
 
 MrpTimer.supportedTypes = ["float"];
 MrpTimer.template = "mrp.MrpTimer";
 
 registry.category("fields").add("mrp_timer", MrpTimer);
+registry.category("formatters").add("mrp_timer", formatMinutes);
```
[the remark this answers is not in the document]
Comment (ajf-odoo) [reply addons/mrp/static/src/widgets/timer.js:90]: not really needed, exported for compliance with other formatters, remove ?

[the remark this answers is not in the document]
Comment (Whenrow) [reply addons/mrp/static/src/widgets/timer.js:90]: yes please. lets keep this patch the simplest

**Comment by Whenrow** [comment]
robodoo r+ rebase-ff

**Pull request closed without merging**